packages/block-library/src/post-excerpt/index.php

```diff
@@ -27,7 +27,8 @@ function render_block_core_post_excerpt( $attributes, $content, $block ) {
 	* wp_trim_words is used instead.
 	*/
 	$excerpt_length = $attributes['excerptLength'];
-	$excerpt        = get_the_excerpt( $block->context['postId'] );
+
+	$excerpt        = apply_filters( 'the_excerpt', get_the_excerpt( $block->context['postId'] ) );
```

Comment:
@CoderAbhinav I think this will result in all the default filters being applied, so it'd be good to make everything still works correctly:
https://github.com/WordPress/wordpress-develop/blob/6106332617128820c0ee5132c34ff1cf6f785d1a/src/wp-includes/default-filters.php#L205-L211

I did some testing personally and couldn't see any issues. I think most of these filters like `convert_smilies` are already run on the content, so there's no change from trunk to this PR.

The change might also need a dev note, as some plugins might be working around the missing filter and reintroducing it could require plugins updating their code.

As long as everything works, then this change seems fine (apart from Mamaduka's request change). There's precedence in that the post content block also does this and has done for a long time:
https://github.com/WordPress/gutenberg/blob/8e048a88af626169423f76136c264234819aef3b/packages/block-library/src/post-content/index.php#L50